src/components/NewTodo/NewTodo.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import Select from '../Select/Select';
+import { NewTodoProps } from '../../constants/proptypes';
+
+import './NewTodo.css';
+
+class NewTodo extends Component {
+  state = {
+    value: '',
+    selectedByDefault: 0,
+    inputError: null,
+    selectError: null,
+  };
+
+  handleChange = (event) => {
```

Comment:
если не используем ивент, можно деструктурировать
```suggestion
  handleChange = ({ target }) => {
```